index.html

```diff
@@ -299,5 +299,58 @@ <h4 class="ico">LET’S CONNECT</h4>
           </section>
         </div>
       </section>
+      <section class="contact">
+        <div class="contact-content">
+        <h1>Contact Me</h1>
+          <p>
+            If you have an application you are interested in developing, a feature
+            that you need built or a project that needs coding. I’d 
+            love to help with it
+          </p>
+        <form action="https://formspree.io/f/mqknjqvv" method="post">
+          <ul class="form-sections">
+            <li>
+              <label for="name"></label>
+              <input
+                type="text"
+                id="name"
+                name="user_name"
+                maxlength="30"
+                placeholder="your name "
+                required
+                class="form-input"
+              />
+            </li>
+            <li>
+              <label for="mail"></label>
+              <input
+                type="email"
+                id="mail"
+                name="user_mail"
+                maxlength="30"
```

Comment:
- [ ] Kindly remove the max length for email input, please implement this as this was requested by the previous reviewer, There should be no max length specified for email 👍🏿